CARITAS-243_Caritas-Rework-WP21-Remove-legacy-code_groupchatv1_Patric-Dosch

patric-dosch-vi opened the pull request.

patric-dosch-vi committed:
chore: remove chatV1 feature

Refs: CARITAS-243

vi-sudo commented:
Task linked: [CARITAS-243 [Caritas Rework] WP2.1: Remove legacy code](https://app.clickup.com/t/6656196/CARITAS-243)

patric-dosch-vi committed:
fix: remove feedbackGroupId from Session

This was removed, but a merge conflict was resolved incorrectly, therefore it was back in the code. While working on the code removals, I actually deleted the columns (the migrations were never executed), which triggered the error.

Refs: CARITAS-243
